site/cds_rdm/templates/semantic-ui/invenio_notifications/email/community-submission.submit.jinja

```diff
@@ -0,0 +1,58 @@
+{% set submission_request = notification.context.request %}
+{% set community = submission_request.receiver %}
+{% set creator = submission_request.created_by %}
+{% set record = submission_request.topic %}
+{% set request_id = submission_request.id %}
+{% set recipient_full_name = recipient.data.profile.full_name %}
+{% set help_url = "https://cern.service-now.com/service-portal?id=service_element&name=CDS-Service" %}
```

Comment:
Can you please put this in `config.py`, so that it can be re-used also in other parts of the sites and easily updated in the future?